study-frontend/src/Services/Authentication/Authentication.ts

```diff
@@ -10,21 +14,11 @@ export const Authentication = async ({email, password} : Props) => {
 
 
     try {
-      
-        const metadata = {
-            email : email,
-            password : password
-
-        }; 
-        const response = await fetch('http://localhost:8080/api/v1/auth/authentication',{
-                method : 'POST',
-                headers : {
-                    'Content-Type' : "application/json"
-                },
-                body : JSON.stringify(metadata)
-            }
-        ); 
-        const data = await response.json();
+        const response = await axiosInstance.post('http://localhost:8080/auth/authentication',{
+            email: email,
```

Comment:
Make sure to add your backend endpoint to env file and move the routes to constants